FMT.ipynb

```diff
@@ -0,0 +1,908 @@
+      "HTTP request sent, awaiting response... 302 FOUND\n",
+      "Location: https://s3-us-west-2.amazonaws.com/qiime2-data/2020.2/tutorials/fmt/fmt-tutorial-demux-2-1p.qza [following]\n",
+      "--2020-04-15 09:31:05--  https://s3-us-west-2.amazonaws.com/qiime2-data/2020.2/tutorials/fmt/fmt-tutorial-demux-2-1p.qza\n",
+      "Resolving s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)... 52.218.216.56\n",
+      "Connecting to s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)|52.218.216.56|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 935392 (913K) [binary/octet-stream]\n",
+      "Saving to: ‘/home/user/qiime2-moving-pictures-tutorial//fmt-tutorial-demux-2.qza’\n",
+      "\n",
+      "/home/user/qiime2-m 100%[===================>] 913.47K   490KB/s    in 1.9s    \n",
+      "\n",
+      "2020-04-15 09:31:07 (490 KB/s) - ‘/home/user/qiime2-moving-pictures-tutorial//fmt-tutorial-demux-2.qza’ saved [935392/935392]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -O $workdir/\"fmt-tutorial-demux-2.qza\" \\\n",
+    "  \"https://data.qiime2.org/2020.2/tutorials/fmt/fmt-tutorial-demux-2-1p.qza\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Importing data as a qiime2 artifact\n",
+    "\n",
+    "All data that is used as input to QIIME 2 is in form of QIIME 2 artifacts, which contain information about the type of data and the source of the data. The first thing we need to do is import these sequence data files into a QIIME 2 artifact.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample_metadata = qiime2.Metadata.load(workdir+'/sample-metadata.tsv')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fmt_tutorial_demux_1 = qiime2.Artifact.load(workdir+'/fmt-tutorial-demux-1.qza')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fmt_tutorial_demux_2 = qiime2.Artifact.load(workdir+'/fmt-tutorial-demux-2.qza')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sequence quality control\n",
+    "\n",
+    "We’ll begin by performing quality control on the demultiplexed sequences using DADA2, but this time we’ll run the `denoise-single` command on each set of demultiplexed sequences individually. Again, we’ll want to start by visualizing sequence quality for some of the samples in each run. When we run denoise-single, we need to use the same values for `--p-trunc-len` and `--p-trim-left` for both runs, so when looking at the visualizations that result from these two commands, think about what values would make sense for these parameters for both commands."
```

Comment:
please add a link to DADA2, I will not post this comment later, but it might be useful for other users if they could relate to resources directly, so if you find adding hyperlinks as in the CLI tutorial useful, please do it